app/javascript/components/TopicSectionSuggestion/index.jsx

```diff
@@ -62,9 +62,9 @@ const TopicSectionSuggestion = ({
             )
           ) :null
         }
-        { amountComplete == 5 ? (
+        { amountComplete == 6 ? (
```

Comment:
Now we have our array we can use that to check everything's complete 😄 If we rename it it would be:

```suggestion
        { amountComplete == allTopicResults.length ? (
```